src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -34,11 +36,12 @@
 
 public class GhprbSimpleStatus extends GhprbExtension implements GhprbCommitStatus, GhprbGlobalExtension,
                                GhprbProjectExtension, GhprbGlobalDefault {
+    private static final Logger logger = Logger.getLogger(GhprbSimpleStatus.class.getName());
 
     @Extension
     public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
-
     private final String commitStatusContext;
+    private final boolean showMatrixStatus;
```

Comment:
Since this object will be null on any legacy systems, please make it a Boolean and account for possible null pointers where it is used.